app/components/Card/index.tsx

```diff
@@ -0,0 +1,27 @@
+import {SVGProps} from 'react'
+import Button from '../Button'
+import './card.css'
+
+interface CardProps {
+  title: string
+  description: string
+  impact: string
+  icon: (props: SVGProps<SVGSVGElement>) => JSX.Element
+  action: string
+  className?: string
+}
+
+export default function Card({title, description, impact, icon, action, className}: CardProps) {
+  className = `card bordered ${className}`
+
+  return (
+    <Button action={action} className={className}>
+      <div className="card-icon">{icon({})}</div>
+      <div className="card-content">
+        <h3 className="large-bold padding-bottom-8">{title}</h3>
```

Comment:
this is actually a p tag!
<img width="700" alt="image" src="https://github.com/user-attachments/assets/e87a2614-d088-44f2-8e6c-273d8811401f">